pages/upgrade-guide/4.0.md

```diff
@@ -171,16 +219,31 @@ jQuery 4.0 adds support for [Trusted Types](https://developer.mozilla.org/en-US/
 
 This change was needed in order to add support for [Trusted Types](https://developer.mozilla.org/en-US/docs/Web/API/Trusted_Types_API). It's not entirely clear what real use cases might be affected by this change, but we're listing it here just in case.
 
-#### Important fix: Don't remove HTML comments from scripts
+#### Important fix: Don't remove HTML comments, including `CDATA` sections, from scripts
 
-Browsers can now handle this without jQuery's help. This change is not expected to affect existing code, but is an important fix because jQuery's logic removing those comments was not correctt in 100% of cases.
+Browsers can now handle this without jQuery's help. This change is not expected to affect existing code, but is an important fix because jQuery's logic removing those comments was not correct in 100% of cases.
 
 ### Selector
 
-#### Breaking change: Drop support for legacy pseudos
+#### Important fix: Backport jQuery selection context logic to selector-native
 
-References in Sizzle wiki: https://github.com/jquery/sizzle/wiki#-backwards-compatible-plugins-for-pseudos-with-arguments
+jQuery has long had a minimal selector engine that could be used in place of Sizzle. This engine was missing some logic that Sizzle had for handling the context of a selection. This change backports that logic to the native selector engine, which may affect code that relied on the old behavior. For example, `$div.find( "> *" )` will no longer throw an error when using the native selector engine.Also, given the following HTML:
 
-#### Important fix: Backport jQuery selection context logic to selector-native
+```html
+<div id="parent">
+	<span></span>
+</div>
+```
+
+The following returns 0 results in jQuery 4.0:
```

Comment:
A minor suggestion, feel free to ignore:
```suggestion
The following returns 0 results in jQuery 4.0 even with the selector-native engine:
```

Reply:
I think it's ok because this section is specifically about selector-native.